amd_openvx_extensions/amd_rpp/source/Brightness.cpp

```diff
@@ -21,100 +21,102 @@ THE SOFTWARE.
 */
 
 #include "internal_publishKernels.h"
-#define NUM_OF_DIMS 5
 
-struct BrightnessLocalData
-{
+struct BrightnessLocalData {
     RPPCommonHandle handle;
     rppHandle_t rppHandle;
-    Rpp32u device_type;
+    Rpp32u deviceType;
     Rpp32u nbatchSize;
     RppPtr_t pSrc;
     RppPtr_t pDst;
     vx_float32 *alpha;
     vx_float32 *beta;
-    RpptDescPtr src_desc_ptr;
+    RpptDescPtr srcDescPtr;
     RpptDesc srcDesc;
     RpptDesc dstDesc;
-    RpptDescPtr dst_desc_ptr;
-    RpptROI *roi_tensor_ptr;
+    RpptDescPtr dstDescPtr;
+    void *roiTensorPtr;
+    RpptROI *roiPtr;
     RpptRoiType roiType;
-    Rpp32u layout;
-    size_t in_tensor_dims[NUM_OF_DIMS];
-    size_t out_tensor_dims[NUM_OF_DIMS];
-    vx_enum in_tensor_type;
-    vx_enum out_tensor_type;
+    Rpp32s inputLayout;
+    Rpp32s outputLayout;
+    size_t inputTensorDims[NUM_OF_DIMS];
+    size_t ouputTensorDims[NUM_OF_DIMS];
+    vx_enum inputTensorType;
+    vx_enum outputTensorType;
 #if ENABLE_HIP
     void *pSrc_dev;
     void *pDst_dev;
-    RpptROI *roi_tensor_ptr_dev;
 #endif
 };
 
-static vx_status VX_CALLBACK refreshBrightness(vx_node node, const vx_reference *parameters, vx_uint32 num, BrightnessLocalData *data)
-{
+static vx_status VX_CALLBACK refreshBrightness(vx_node node, const vx_reference *parameters, vx_uint32 num, BrightnessLocalData *data) {
```

Comment:
We are not using parameter num anywhere else in this function
Can this be removed?

Reply:
Since all nodes follow this way we cannot remove the variable

Reply:
Since all augmentations follows this standard we cannot make the change here. It is required